scripts/build.js

```diff
@@ -0,0 +1,31 @@
+const proxyquire =  require('proxyquire');
```

Comment:
Do we need to comment in this file why we are doing this (for customers' understanding)? Or will we document this in the docs?

Reply:
I think this is a detail customer don't necessarily need to be aware of. 

I moved the webpack configuration changes into another file. That should make it more obvious what those 20 lines do.

Reply:
> I think this is a detail customer don't necessarily need to be aware of.

Well, they will be handed this code and asked to maintain it in their app. I think they do need to understand why the files need to be named like they are, otherwise they might think they can just delete this from their project.